src/CellData/DiracDeltas.jl

```diff
@@ -76,3 +79,33 @@ end
 function get_triangulation(d::DiracDelta)
   d.Γ
 end
+
+# For handling DiracDelta at a generic Point in the domain #
+
+function DiracDelta(x::Point{D,T}, model::DiscreteModel{D}) where {D,T}
+  trian = Triangulation(model)
+  cache1 = _point_to_cell_cache(KDTreeSearch(),trian)
+  cell = _point_to_cell!(cache1, x)
+  point_grid = UnstructuredGrid([x])
+  point_model = UnstructuredDiscreteModel(point_grid)
+  point_trian = Triangulation(point_model)
+  dx = Measure(point_trian,1)
+  DiracDelta{0,NotGridEntity}(point_trian,dx)
+end
+
+function DiracDelta(v::Vector{Point{D,T}},model::DiscreteModel{D}) where {D,T}
+  trian = Triangulation(model)
+  cache1 = _point_to_cell_cache(KDTreeSearch(),trian)
+  cell = map(x -> _point_to_cell!(cache1, x), v)
+  point_grid = UnstructuredGrid(v)
+  point_model = UnstructuredDiscreteModel(point_grid)
+  point_trian = Triangulation(point_model)
+  dx = Measure(point_trian,1)
+  DiracDelta{0,NotGridEntity}(point_trian,dx)
+end
+
+function evaluate!(cache,d::DiracDelta{0,NotGridEntity},f::CellField)
```

Comment:
why do we need this specialized version of `evaluate!`. Doesn't the generic one work straight away?

Reply:
No.. it results in quadrature and `FEFunction/CellField` not having the same/compatible background triangulation error

Reply:
Ok. I now understand. The integrand is defined on a domain which is not (currently) compatible with the domain of the measure. But this may mean that there is a compatibility rule among triangulations that is somehow missing, because why these domains should not be compatible?

Reply:
This is just a though ... it you have the impression that this overly complicates thing, we may stick into the current workaround.

Reply:
Initially I tried to make it work with the same triangulations but it wasn't generalising properly due to `CellQuadrature` constructing extra-cells of `#undef` values in `DomainContribution`. 

The problem occurs at @check is_change_possible(trian_f,trian_x) msg in the integrate method (https://github.com/gridap/Gridap.jl/blob/2a4921b473fe113b9d4b6168b8728a7a708f7bc6/src/CellData/CellQuadratures.jl#L142)

Reply:
> But this may mean that there is a compatibility rule among triangulations that is somehow missing, because why these domains should not be compatible?
Yes I was wondering about this as well.. will have to dig deeper